shell/server/pre-meteor.js

```diff
@@ -43,6 +43,12 @@ function isSandstormShell(hostname) {
   return (hostname === HOSTNAME || (DDP_HOSTNAME && hostname === DDP_HOSTNAME));
 }
 
+function hasPseudoTLD(hostname) {
+  // Returns true if this hostname ends in one of the major pseudo-TLDs.
+  let tld = hostname.split(".").pop();
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let it looks like `let` is an ECMAScript 6 thing, which I wish we could use yet, but I guess we can't yet.